games/d.yaml

```diff
@@ -549,7 +550,8 @@
 
 - name: Dungeon Keeper
   meta:
-    genre: [simulation,RTS]
+    genre: [Strategy, RTS]
```

Comment:
No point in having both Strategy and RTS here. Also this is definitely a "dungeon simulation" game.